File: 1.Interview_practce/Array/Rmv_duplicate.js
function duplicate(){
    const arr=[1,2,1,3,2,6,5,4]
    const arr2=[];
    for(let i=0;i<arr.length;i++){
    let found=false;

        for(let j=0;j<arr2.length;j++){
            if(arr[i]===arr2[j]){
                found=true
                break
            }
        }
        if(!found){
            arr2.push(arr[i])
        }
    }
    console.log(arr2)
}

duplicate()

File: 1.Interview_practce/Array/Rmv_duplicate.java
public class Rmv_duplicate {
    public static void main(String[] args) {
        int[] arr={1,2,6,1,7,9,5,4,7,9};
        for(int i=0;i<arr.length-1;i++){
            for(int j=i+1;j<arr.length;j++){
            if(arr[i]==arr[j]){
                System.out.println(arr[j]);
                break;
        
            }
        }
        }
    }
}
